its-live/README.md

```diff
@@ -0,0 +1,56 @@
+# Inter-mission Time Series of Land Ice Velocity and Elevation (ITS-LIVE) dataset
+
+The Inter-mission Time Series of Land Ice Velocity and Elevation (ITS-LIVE) project has a singular mission: to accelerate ice sheet and glacier research by producing globally comprehensive, high resolution, low latency, temporally dense, multi-sensor records of land ice and ice shelf change, while minimizing barriers between the data and the user.
+
+ITS-LIVE data consists of:
+* NetCDF Level-2 scenes-pair ice flow products posted to a standard 120 m grid derived from Landsat 4/5/7/8/9 and Sentinel-2 optical scenes, and Sentinel-1 SAR scenes.
+* A set of Zarr datacubes containing all scene pair data cloud-optimized for time-series analysis, with a suite of user tools available to effectively query the data.
+* A suite of Level 3 products, including: quarterly and annual ice-sheet and regional velocity mosaics as Cloud-Optimized GeoTIFFs
+
+
+## Accessing the ITS-LIVE data
+
+The ITS-LIVE data is all stored is the public `its-live-data` AWS S3 bucket, which is located in the `us-west-2` (Oregon) AWS Region. 
+
+```shell
+aws s3 ls s3://its-live-data/
+```
+
+> [CAUTION!]
+> There are over 1 billion objects in the `its-live-data` bucket. Recursively listing the whole bucket is **not** recommended.
+
+The data is organized under a collection of prefixes (folders) to ease access, which are described below. We've **bolded** the prefixes which are typically of the most interest to users:
+
+* `L7_PV_fix/`: 
+* `NSIDC/`: 
+* `Test/`: 
+* `autorift_parameters/`: 
+* `catalog_geojson/`: 
+* `catalog_geojson_latest/`: 
+* `catalog_geojson_original/`: 
+* `composites/`: 
+* `datacubes/`: 
+* `documentation/`: 
+* `elevation/`: 
+* `height_change/`: 
+* `ice_masks/`: 
+* `ice_shelf/`: 
+* `isce_autoRIFT/`: 
+* `month-data-logs/`: 
+* `mosaics/`: 
+* `qgis_project/`: 
+* `rgb_mosaics/`: 
+* `test/`: 
+* `test_datacubes/`: 
+* **`vel_web_tiles/`**: 
+* **`velocity_image_pair/`**: 
+* **`velocity_mosaic/`**: 
```

Comment:
@alex-s-gardner, @mliukis, I'd like to explain (broadly) what each prefix is for in the bucket. Are we going to keep all of them? I expect many aren't going to be particularly interesting to users, so I'd like to bold the ones users will care about. 

Alternatively, we can just list the ones users should care about, or break it into two sections.

Reply:
@jhkennedy The following folders are not end user intended:

* NSIDC - all corrected v01 data products that we ingested into NSIDC
     @alex-s-gardner should we remove all of them on our side? May be good to keep them as a reference until we are done with v02 ingest into NSIDC?
* test_datacubes - this is our test data (chunking, verification of new data, etc.)
* catalog_geojson_original - removed
* catalog_geojson_latest - removed
* catalog_geojson - @alex-s-gardner do we want to expose catalog geojsons to the user?

These look like some staged data for temporary use (?):
* Test:
    2023-07-24 09:31:04  209641057 N52W175.nc
* L7_PV_fix:
    2023-07-17 10:18:55  238194861 l7_pvalue_comp_with_fix.parquet
* month-data-logs: @alex-s-gardner there are some monthly logs from 2022. Do we still need them?
* test - some test data

Reply:
Going mark this resolved and will track what data is going where in: https://github.com/ASFHyP3/OpenData/issues/10